javascript/javascript3/week2/app.js

```diff
@@ -0,0 +1,69 @@
+let exchangeRates;
+
+const getExchangeData = async () => {
+  const res = await fetch("https://open.er-api.com/v6/latest/USD");
+  const data = await res.json();
+  exchangeRates = data.rates;
+  const currenciesList = Object.keys(exchangeRates);
+  console.log(exchangeRates);
+  console.log(currenciesList);
+
+  const currencyFromDropdown = document.getElementById("currency-from");
+  const currencyToDropdown = document.getElementById("currency-to");
+
+  currenciesList.forEach((currency) => {
+  const option = document.createElement("option");
+  option.text = currency;
+  if (currency === "EUR") {
+    option.setAttribute("selected", true);
+  }
+  currencyFromDropdown.add(option);
+  });
+
+  currenciesList.forEach((currency) => {
+  const option = document.createElement("option");
+  option.text = currency;
+  if (currency === "DKK") {
+    option.setAttribute("selected", true);
+  }
+  currencyToDropdown.add(option);
+  });
+  
+  };
+  getExchangeData();
```

Comment:
For functions like this it is better to add them to the window.onload listener. That way you can be sure they will be called at the right time, i.e. once the page is fully loaded. 

See https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event